src/pybullet_blocks/planning_models/action.py

```diff
@@ -51,6 +50,23 @@
 Robot = Variable("?robot", robot_type)
 Obj = Variable("?obj", object_type)
 Surface = Variable("?surface", object_type)
+
+
+def add_clear_effects(
```

Comment:
sorry but can we separate out all the code in this PR that isn't directly related to the collision checking in the step function 🙏 thanks :)